pages_builder/push_to_github_pages.sh

```diff
@@ -0,0 +1,35 @@
+current_branch=$(git rev-parse --abbrev-ref HEAD)
```

Comment:
Addressed in e40c5aad and at https://github.com/alphagov/digitalmarketplace-frontend-toolkit/commit/9bbe884f1a0554232b9705fb774281bbda5b8662#diff-a1fc7d98932ecd188caf8f5fd6414159R1